src/theory/quantifiers/sygus/sygus_grammar_norm.cpp

```diff
@@ -208,31 +210,29 @@ void SygusGrammarNorm::TypeObject::addConsInfo(SygusGrammarNorm* sygus_norm,
   }
 
   std::vector<TypeNode> consTypes;
-  for (const DatatypeConstructorArg& arg : cons)
+  const std::vector<std::shared_ptr<DTypeSelector> >& args = cons.getArgs();
+  for (const std::shared_ptr<DTypeSelector>& arg : args)
```

Comment:
You changed because it is more efficient to do it like this?

Reply:
No, the previous form of iteration is no longer available (we are switching Expr -> Node here), so this was necessary to make it compile.